cms/templates/studio_xblock_wrapper.html

```diff
@@ -89,6 +89,13 @@
                                     <span class="sr">${_("Duplicate")}</span>
                                     </a>
                                 </li>
+
+                                <li class="action-item action-move">
+                                    <a href="#" data-tooltip="${_("Move")}" class="move-button action-button">
```

Comment:
I believe this should be an actual button instead of an a-tag (and they all should be...). At the time that Studio was written, UX commonly used a-tags instead of buttons, but that is incorrect and bad for accessibility.